src/storm-pomdp-cli/storm-pomdp.cpp

```diff
@@ -357,13 +457,19 @@ namespace storm {
                     std::string transformMode = transformSettings.getFscApplicationTypeString();
                     auto pmc = toPMCTransformer.transform(storm::transformer::parsePomdpFscApplicationMode(transformMode));
                     STORM_PRINT_AND_LOG(" done.\n");
-                    pmc->printModelInformationToStream(std::cout);
+                    uint_fast64_t numberOfStates = pmc->getTransitionMatrix().getColumnCount();
+
+
                     if (transformSettings.allowPostSimplifications()) {
                         STORM_PRINT_AND_LOG("Simplifying pMC...");
                         pmc = storm::api::performBisimulationMinimization<storm::RationalFunction>(pmc->template as<storm::models::sparse::Dtmc<storm::RationalFunction>>(),{formula.asSharedPointer()}, storm::storage::BisimulationType::Strong)->template as<storm::models::sparse::Dtmc<storm::RationalFunction>>();
                         STORM_PRINT_AND_LOG(" done.\n");
                         pmc->printModelInformationToStream(std::cout);
                     }
+                    if (transformSettings.isConstantRewardsSet()) {
+                        pmc = makeRewardsConstant<ValueType, DdType>(pmc);
+                    }
+                    pmc->printModelInformationToStream(std::cout);
```

Comment:
We print the model here without saying what we just did? I think it would be more consistent to add a STORM_PRINT_AND_LOG (as with simplification) or decide that we only print it once at the very end and remove all earlier printModelInformations (either is fine with me)

Reply:
I'll add STORM_PRINT_AND_LOG